docs/v2.4/reference/config.md

```diff
@@ -65,21 +79,21 @@ to handle installation of the `traffic-agents`.
 
 These are the valid fields for the `images` key:
 
-|Field|Description|Default|
-|---|---|---|
-|`registry`|Docker registry to be used for installing the Traffic Manager and default Traffic Agent. If not using a helm chart to deploy server-side objects, changing this value will create a new traffic-manager deployment when using Telepresence commands. Additionally, changing this value will update installed default `traffic-agents` to use the new registry when creating a new intercept.|docker.io/datawire|
-|`agentImage`|$registry/$imageName:$imageTag to use when installing the Traffic Agent. Changing this value will update pre-existing `traffic-agents` to use this new image. * the `registry` value is not used for the `traffic-agent` if you have this value set *||
-|`webhookRegistry`|The container $registry that the [Traffic Manager](../cluster-config/#mutating-webhook) will use with the `webhookAgentImage` *This value is only used if a new traffic-manager is deployed*||
-|`webhookAgentImage`|The container image that the [Traffic Manager](../cluster-config/#mutating-webhook) will use when installing the Traffic Agent in annotated pods *This value is only used if a new traffic-manager is deployed*||
+| Field               | Description                                                                                                                                                                                                                                                                                                                                                                                    | Default              |
```

Comment:
I know they are all strings, but just for the sake of uniformity, should we include a type column here as well?

Reply:
Done.